arch/cpu/nrf/dev/slip-arch.c

```diff
@@ -31,17 +31,26 @@
 /*---------------------------------------------------------------------------*/
 #include "contiki.h"
 #include "dev/slip.h"
-#include "dev/uarte-arch.h"
+#include "uarte-arch.h"
+#include "usb.h"
 /*---------------------------------------------------------------------------*/
 void
 slip_arch_writeb(unsigned char c)
 {
+#if PLATFORM_DBG_CONF_USB
+  usb_write(&c, sizeof(c));
```

Comment:
This, as well as dbg-arch.c, will end up calling flush for every single byte.

I suggest handling this in the same way as the cc2538 and nrf52840 platforms:
https://github.com/contiki-ng/contiki-ng/blob/343e25144228fc6997cf6d02f4ffd4b0b180764f/arch/cpu/nrf52840/slip-arch.c#L57-L60

Reply:
Thank you. Done.